_posts/2016-9-2-jade-release.markdown

```diff
@@ -0,0 +1,67 @@
+Hello everyone,
+
+We have a number of updates for MoveIt!
+
+The 1st official MoveIt! release into ROS Jade has finally happened
+===================================================================
+
+If you're running your robot packages with ROS Jade, and/or if you want to try the newest MoveIt!, start at [moveit.ros.org/install](http://moveit.ros.org/install/)
+
+Among numerous bug-fixes, there are quite a number of new features included:
+
+- [You can stop a trajectory while it's being executed via RViz](https://github.com/ros-planning/moveit_ros/pull/713)
+  [Video](https://www.youtube.com/watch?v=XEU-wVHUvgI&feature=youtu.be)
+
+- [Capability to execute trajectory with a ROS action](https://github.com/ros-planning/moveit/pull/60)
+
+- [Maximum acceleration scaling factor](https://github.com/ros-planning/moveit_core/pull/273)
+
+- [Enables setting optimization objectives from ompl_planning.yaml](http://docs.ros.org/indigo/api/moveit_tutorials/html/doc/ompl_interface_tutorial.html?highlight=ompl)
+
+- [Velocity & acceleration values to cartesian trajectory](https://github.com/ros-planning/moveit_ros/pull/735)
+
+- [ApplyPlanningSceneService capability](https://github.com/ros-planning/ros-planning/moveit_ros/pull/686)
```

Comment:
bad URL, travis is failing because of it

@mintar html-proofer is awesome, thank you so much!